Core/src/main/java/com/paypal/android/core/analytics/AnalyticsClient.kt

```diff
@@ -0,0 +1,30 @@
+package com.paypal.android.core.analytics
+
+import android.util.Log
+import com.paypal.android.core.Http
+import com.paypal.android.core.HttpRequestFactory
+import java.util.UUID
+
+internal class AnalyticsClient(
+    private val deviceInspector: DeviceInspector,
+    private val http: Http,
+    private val httpRequestFactory: HttpRequestFactory
+) {
+
+    internal suspend fun sendAnalyticsEvent(name: String) {
+        val timestamp = System.currentTimeMillis()
+
+        val analyticsEventData =
+            AnalyticsEventData(name, timestamp, sessionId, deviceInspector.inspect())
+        val httpRequest = httpRequestFactory.createHttpRequestForAnalytics(analyticsEventData)
+
+        val response = http.send(httpRequest)
+        if (!response.isSuccessful) {
+            Log.d("[PayPal SDK]", "Failed to send analytics: ${response.error?.message}")
+        }
+    }
+
+    companion object {
+        private val sessionId = UUID.randomUUID().toString()
```

Comment:
this is equal to `static` so we will have only one session ID as long as the merchant app is still in memory